core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,50 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
+@media(max-width:600px){
+ .oppia-navbar-brand-name{
```

Comment:
No. Spaces for everything, but use 2 space indentation, and add a single space before each '{'.